packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -228,10 +228,12 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
       </Button.Group>
       <BannerExtension />
       <BannerClaims />
+
+      {summary}
+
```

Comment:
```suggestion
      <Summary balance={balances.summary} />
```

The Summary component should be React.memo-ed itself.

Reply:
Done